examples/terraform/basic/files/patch.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
Can we use yaml patch files now? Way easier to understand and maintain...

Reply:
yes, catching that in a follow-up PR. good idea to standardize the patches.